kolibri/plugins/html5_app_renderer/assets/src/vue/index.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+
+  <div v-el:container class="container" allowfullscreen>
+    <icon-button
+      class="btn"
+      :text="inFullscreen ? $tr('exitFullscreen') : $tr('enterFullscreen')"
+      @click="togglefullscreen">
+    </icon-button>
+    <iframe v-el:sandbox class="sandbox" :src="indexpath" sandbox="allow-scripts"></iframe>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+
+    components: {
+      'icon-button': require('kolibri.coreVue.components.iconButton'),
+    },
+
+    props: ['defaultFile'],
+
+    data: () => ({
+      inFullscreen: false,
+    }),
+
+    computed: {
+      indexpath() {
+        return this.defaultFile.storage_url + "index.html";
+      }
+    },
+
+    methods: {
+      togglefullscreen() {
```

Comment:
A library just for that may be overkill, but yeah, I was thinking to suggest that this be refactored out into a core Vue component (full-screenable-content-area or something :grey_question:) that then contains a slot for the actual content area. We may also want to make full-screening of content a standard option across content types (e.g. could be great for exercises), in which case it could be integrated into the base content viewer.